web/src/frontend/classic/components/settings/SettingsPanel.svelte

```diff
@@ -3,89 +3,68 @@
     import SettingItem from './SettingItem.svelte';
     import {
         ContentPanel,
-        ContentPanelValue,
         Key,
         Locale,
         FuzzySearch,
-        FuzzySearchValue,
         Theme,
-        ThemeValue,
         ViewerDoublePage,
-        ViewerDoublePageValue,
         ViewerMode,
-        ViewerModeValue,
         ViewerReverseDirection,
-        ViewerReverseDirectionValue,
     } from '../../stores/Settings';
+    import type { Choice } from '../../../../engine/SettingsManager';
+
+    const ThemeSetting = Theme.setting as Choice;
```

Comment:
Cast will no longer be necessary in case of using the suggested generic approach for `CreateWriteableStore`

Reply:
That did the trick !